src/app/policies/views/PolicyListView.vue

```diff
@@ -11,45 +11,45 @@
       v-slot="{data: policies, error: policyError}: PolicyTypeCollectionSource"
       :src="`/*/policy-types`"
     >
+      <LoadingBlock v-if="policies === undefined" />
+
       <ErrorBlock
-        v-if="policyError"
+        v-else-if="policyError"
         :error="policyError"
       />
-      <LoadingBlock v-else-if="policies === undefined" />
+
       <EmptyBlock v-else-if="policies.policies.length === 0" />
 
-      <template
-        v-else
-      >
+      <template v-else>
         <template
-          v-for="selected in [policies.policies.find(item => item.path === route.params.policyPath) ?? policies.policies[0]]"
-          :key="selected.path"
+          v-for="currentPolicyType in [policies.policies.find((item) => item.name === route.params.policyType) ?? policies.policies[0]]"
```

Comment:
Is `currentPolicyType` the policyType from the current URL, or it is the one I've `selected`?

i.e. I'm not sure of the need to change this.

Reply:
It’s the same value with a better variable name.